tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: request-and-upload-signature
   labels:
-    app.kubernetes.io/version: "1.0.0"
+    app.kubernetes.io/version: "1.1.0"
```

Comment:
This is a breaking change, since the parameter names changed, so should be 2.0.0.